### ISSUE TITLE
🐛 Bug: ERROR [i] Knex: Timeout acquiring a connection. The pool is probably full.

### ISSUE THREAD

**Issue by emre1e** [nocodb/nocodb#6272]
### Please confirm if bug report does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

slow performance
Docker Logs 100000 entries:

[Nest] 9  - 08/30/2023, 1:15:46 PM   ERROR [i] Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?
KnexTimeoutError: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?
    at Client_PG.acquireConnection (/usr/src/app/node_modules/knex/lib/client.js:307:26)
    at runMicrotasks (<anonymous>)
    at runNextTicks (node:internal/process/task_queues:61:5)
    at listOnTimeout (node:internal/timers:528:9)
    at processTimers (node:internal/timers:502:7)
    at async Runner.ensureConnection (/usr/src/app/node_modules/knex/lib/execution/runner.js:287:28)
    at async Runner.run (/usr/src/app/node_modules/knex/lib/execution/runner.js:30:19)
    at async Function.get (/usr/src/app/docker/main.js:1:865913)
    at async t.BaseModelSqlv2.selectObject (/usr/src/app/docker/main.js:1:215779)
    at async t.BaseModelSqlv2.hmList (/usr/src/app/docker/main.js:1:202383)
KnexTimeoutError: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?
    at Client_PG.acquireConnection (/usr/src/app/node_modules/knex/lib/client.js:307:26)
    at runMicrotasks (<anonymous>)
    at runNextTicks (node:internal/process/task_queues:61:5)
    at listOnTimeout (node:internal/timers:528:9)
    at processTimers (node:internal/timers:502:7)
    at async Runner.ensureConnection (/usr/src/app/node_modules/knex/lib/execution/runner.js:287:28)
    at async Runner.run (/usr/src/app/node_modules/knex/lib/execution/runner.js:30:19)
    at async Function.get (/usr/src/app/docker/main.js:1:865913)
    at async t.BaseModelSqlv2.selectObject (/usr/src/app/docker/main.js:1:215779)
    at async t.BaseModelSqlv2.hmList (/usr/src/app/docker/main.js:1:202383) {
  sql: undefined,
  bindings: undefined
}
KnexTimeoutError: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?
    at Client_PG.acquireConnection (/usr/src/app/node_modules/knex/lib/client.js:307:26)
    at runMicrotasks (<anonymous>)
    at runNextTicks (node:internal/process/task_queues:61:5)
    at listOnTimeout (node:internal/timers:528:9)
    at processTimers (node:internal/timers:502:7)
    at async Runner.ensureConnection (/usr/src/app/node_modules/knex/lib/execution/runner.js:287:28)
    at async Runner.run (/usr/src/app/node_modules/knex/lib/execution/runner.js:30:19)
    at async Function.get (/usr/src/app/docker/main.js:1:865913)
    at async t.BaseModelSqlv2.selectObject (/usr/src/app/docker/main.js:1:215779)
    at async t.BaseModelSqlv2.mmList (/usr/src/app/docker/main.js:1:205285) {
  sql: undefined,
  bindings: undefined
}

### Desired Behavior

knex version bugged?

### Project Details

docker pg 0.109.6

**Comment by dstala** [comment]
apologies for delayed follow up on this. issue is with an older version. @emre1e please please reopen this issue if you still face this issue with our latest release v0.202.5;

**Comment by jackyu1996** [comment]
I had suffered from this problem too and my finding was that: when using nftables, I adopted the **drop policy** in the **forward chain** by default, and this has led to database connection timeout (I have tested both postgres and mysql) for docker deployments. By **removing the drop policy**, it works as expected. However, to be more secure, more configuration might be needed for nftables to work well with docker.
